### DISCUSSION TITLE
Music visualizer

### DISCUSSION THREAD

**Discussion by Dubstop26789** [rocksdanister/lively#1837] [General]
It would be cool if the there were more live music visualizers like in the Music TV. I personally would love a visualizer in the style of either Simple System or The Hill

**Comment by Paulchemouni** [comment]
I would also love if there was a visualizer who would just put the album art without anything else as a wallpaper, with a blur around, like the GNS app does for android ([https://play.google.com/store/apps/details?id=com.mejmun.goodnightspotify&hl=fr&gl=US](url))

**Comment by rocksdanister** [comment]
Some sample projects are here:
https://github.com/rocksdanister/audio-visualizer-wallpaper

To make your own:
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data

**Comment by iambecomingrich** [comment]
I know this is old but I use this one:
https://github.com/rocksdanister/audiorbits

There is a very slight delay but it's only noticeable if you are staring at it for long periods of times
